packages/SwingSet/test/upgrade/test-upgrade.js

```diff
@@ -686,20 +663,18 @@ test('non-durable exports are abandoned by upgrade of non-liveslots vat', async
 
   // Export two objects from exporter to observer,
   // one to be held strongly and the other weakly.
-  const observer = await run('getVatRoot', [
-    { name: 'observer', rawOutput: true },
+  const observer = await messageToVat('bootstrap', 'getVatRoot', 'observer');
+  const strongObj = await messageToVat('exporter', 'exportFakeObject');
+  await messageToVat('exporter', 'syscall-send', observer, 'acceptImports', [
+    strongObj,
   ]);
```

Comment:
This kind of call really would benefit from better structure delineation... it takes a lot of cogitation to parse this as an invocation of the "exporter" vat's "syscall-send" method with three arguments, as opposed to e.g. `V('exporter')['syscall-send'](observer, 'acceptImports', [strongObj])`. Although it _is_ a specialized case that could be handled with a local function if things get out of hand.

Reply:
I think something like the `E`/`EV` notation would really help here.